tests/test_makeps3iso.py

```
@@ -181,6 +335,40 @@ async def test_plan_job_rejects_output_inside_source(tmp_path):
+):
+    # A SKIP request whose output already exists must short-circuit with
+    # OUTPUT_EXISTS *without* running the recursive PS3 safety walk — otherwise a
+    # batch skip over a large already-converted library stats every tree for
+    # nothing. The worker still revalidates the tree for jobs that actually run.
+    _confine_to_volume(monkeypatch, tmp_path)
+    folder = _make_ps3_folder(tmp_path / "MyGame")
+    (tmp_path / "MyGame.iso").write_bytes(b"iso")  # existing sibling output
+
+    walked = False
+
+    def _tracking_walk(path):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_tracking_walk`

Add return type annotation: `bool`

(ANN202)